spec/interators/comments/create/prepare_params_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Comments::Create::PrepareParams do
+  subject(:context) { described_class.call(user: user, article: article, params: params) }
```

Comment:
давай основной метод который тестим называть так же как он называется, в данном случае call, не context

```suggestion
  subject(:call) { described_class.call(user: user, article: article, params: params) }
```